src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/AccountGlobal.xml

```diff
@@ -10,14 +10,238 @@
+        <ref bean="AccountGlobal-contractsAndGrantsAccountResponsibilityId"/>
+        <ref bean="AccountGlobal-invoiceFrequencyCode"/>
+        <ref bean="AccountGlobal-invoiceTypeCode"/>
+        <ref bean="AccountGlobal-costShareForProjectNumber"/>
+        <ref bean="AccountGlobal-removeAccountExpirationDate"/>
+        <ref bean="AccountGlobal-removeContinuationChartAndAccount"/>
+        <ref bean="AccountGlobal-financialIcrSeriesIdentifier"/>
+        <ref bean="AccountGlobal-everify"/>
+        <ref bean="AccountGlobal-removeIncomeStreamChartAndAccount"/>
       </list>
     </property>
+    <property name="relationships">
+      <list merge="true">
+        <bean parent="RelationshipDefinition" p:objectAttributeName="accountPhysicalCampus">
+			<property name="primitiveAttributes">
+		    	<list>
+		        	<bean parent="PrimitiveAttributeDefinition" p:sourceName="accountPhysicalCampusCode" p:targetName="code" />
+		        </list>
+		    </property>
+		</bean> 
+      </list>
+      </property>
   </bean>
+  
 
   <!-- Attribute Definitions -->
   <bean id="AccountGlobal-majorReportingCategoryCode" parent="AccountGlobal-majorReportingCategoryCode-parentBean"/>
   <bean id="AccountGlobal-majorReportingCategoryCode-parentBean" abstract="true" parent="MajorReportingCategory-majorReportingCategoryCode">
     <property name="required" value="false"/>
   </bean>
   
+	<bean id="AccountGlobal-accountPhysicalCampusCode" parent="AccountGlobal-accountPhysicalCampusCode-parentBean" />
+	<bean id="AccountGlobal-accountPhysicalCampusCode-parentBean" abstract="true"
+		parent="Account-accountPhysicalCampusCode">
+	</bean>
+	
+	<bean id="AccountGlobal-accountEffectiveDate" parent="AccountGlobal-accountEffectiveDate-parentBean" />
+	<bean id="AccountGlobal-accountEffectiveDate-parentBean" abstract="true"
+		parent="Account-accountEffectiveDate">
+	</bean>
+	
+	<bean id="AccountGlobal-accountOffCampusIndicator" parent="AccountGlobal-accountOffCampusIndicator-parentBean" />
+	<bean id="AccountGlobal-accountOffCampusIndicator-parentBean" abstract="true"
+		parent="Account-accountOffCampusIndicator">
+	</bean>
+	
+	<bean id="AccountGlobal-closed" parent="AccountGlobal-closed-parentBean" />
+	<bean id="AccountGlobal-closed-parentBean" abstract="true"
+		parent="Account-closed">
+	</bean>
+	
+	<bean id="AccountGlobal-accountTypeCode" parent="AccountGlobal-accountTypeCode-parentBean" />
+	<bean id="AccountGlobal-accountTypeCode-parentBean" abstract="true"
+		parent="Account-accountTypeCode">
+	</bean>
+	
+	<bean id="AccountGlobal-appropriationAccountNumber" parent="AccountGlobal-appropriationAccountNumber-parentBean" />
+	<bean id="AccountGlobal-appropriationAccountNumber-parentBean" abstract="true" parent="AppropriationAccount-appropriationAccountNumber">
+		<property name="required" value="false"/>
+	</bean>
+		
+	<bean id="AccountGlobal-accountsFringesBnftIndicator" parent="AccountGlobal-accountsFringesBnftIndicator-parentBean" />

```

Comment:
As per my earlier comment, I would suggest updating these boolean fields (except possibly the "remove..." fields) to use the same control override as "AccountGlobal-pendingAcctSufficientFundsIndicator"